mautic/mautic
# Allow composer scripts to use php-cs-fixer configuration to allow multiple files at once

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🚫 
| New feature/enhancement? (use the a.x branch)      | enhancement
| Deprecations?                          | 🚫 
| BC breaks? (use the c.x branch)        | 🚫 
| Automated tests included?              | 🚫  
| Related user documentation PR URL      | 🚫  
| Related developer documentation PR URL | 🚫  
| Issue(s) addressed                     | 🚫  



#### Description:
Whenever `composer fixcs` is executed, it does not use configurations mentioned in `.php-cs-fixer.php`. This PR allows composer scripts to use the php-cs configurations explicitly.
The advantage of this is mentioned in the comment below: https://github.com/mautic/mautic/pull/10624#issuecomment-1049763294



#### Steps to test this PR:
1. Modify any PHP file such that there will be CS fixed to be done, like add spaces between assignment operator.
2. Run `composer fixcs`.
3. Observe that, the composer script does not use `.php-cs-fixer.php` configuration.


1. Load up [this PR](https://mautibox.com)
2. Follow steps to test this PR.
3. Observe `.php-cs-fixer.php` configuration being used and file is CS fixed.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10624"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
